addons/hr_expense/models/hr_expense.py

```diff
@@ -538,6 +538,7 @@ def _get_account_move_line_values(self):
                 'currency_id': expense.currency_id.id,
                 'expense_id': expense.id,
                 'partner_id': partner_id,
+                'exclude_from_invoice_tab': True,
```

Comment:
will be a mess in migration..

Reply:
not that much of a problem: old expenses will have the old behaviour and won't show in_payment state as before, while the new ones will work as per the new code.

Looks fine to me